README.md

```diff
@@ -70,79 +70,76 @@ We have three aliases for product versions;
   the JetBrains Beta channel.
   * `under-dev` is the IDE version we are currently working towards supporting.
 
-Th current corresponding IDE versions of these aliases are:
-
-| Product-version alias            | Version |
-| ---------------------------------|:-------:|
-| intellij-ue-oss-stable           | 2020.3  |
-| intellij-ue-oss-beta             | 2020.3  |
-| intellij-ue-oss-under-dev        | 2021.1  |
-| intellij-oss-stable              | 2020.3  |
-| intellij-oss-beta                | 2020.3  |
-| intellij-oss-under-dev           | 2021.1  |
-| android-studio-oss-stable        | 2020.3  |
-| android-studio-oss-beta          | 2020.3  |
-| android-studio-oss-under-dev     | 2021.1  |
-| clion-oss-stable                 | 2020.3  |
-| clion-oss-beta                   | 2020.3  |
-| clion-oss-under-dev              | 2021.1  |
+The current corresponding IDE versions of these aliases can be found [here](./intellij_platform_sdk/build_defs.bzl#L31).
 
 ## Contributions
 
 We welcome contributions to support new IDE versions. However, to make
 the review process faster and easier, we recommend the following:
 
-  * Try to make changes in smaller patches. A smaller pull request will 
-    make the review faster and more thorough. You are encouraged to have 
-    separate pull requests each focusing on a ceratin incompatible change
+  * We can only accept small pull requests. A smaller pull request would likely have less review
+    comments and hence can get submitted much faster. For us, it is also much easier to integrate
```

Comment:
I used this formulation in my own comment but it sounds a bit long-winded in the documentation. Could we go with something more straight-forward like "Smaller pull requests tend to have less review comments and hence can get submitted much faster. They also tend to conflict less with our internal code base, simplifying the integration for us."?